<launch>

    <node name="my_commander_node" pkg="ucar_commander" type="my_commander_node.py" output="screen">
        <rosparam file="$(find ucar_commander)/config/ucar_commander.yaml" command="load" />
    </node>

</launch>